From: Danny Kukawka <dkukawka@suse.de>
Date: Thu, 5 Jun 2014 00:07:40 +0200
Subject: [PATCH] init-ceph: add xfs support

Merge distro patches to git.

This was ceph-init-ceph.add.xfs.support.diff

(cherry picked from commit b61e60686ad17bd0e842c9f87040344b70d04a59)
---
 src/init-ceph.in | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/src/init-ceph.in b/src/init-ceph.in
index fb997c1..ffbeda3 100644
--- a/src/init-ceph.in
+++ b/src/init-ceph.in
@@ -139,12 +139,20 @@ case $1 in
     --btrfs | --fsmount)
 	    dofsmount=1
 	    ;;
+    --xfs)
+	    doxfs=1
+	    dobtrfs=0
+	    ;;
     --nobtrfs | --nofsmount)
 	    dofsmount=0
 	    ;;
     --btrfsumount | --fsumount)
 	    dofsumount=1
 	    ;;
+    --xfsumount)
+	    doxfsumount=1
+	    dobtrfsumount=0
+	    ;;
     --conf | -c)
 	    [ -z "$2" ] && usage_exit
 	    options="$options $1"
